src/firebolt/async_db/_types.py

```diff
@@ -143,88 +143,46 @@ def __eq__(self, other: object) -> bool:
         return other.precision == self.precision and other.scale == self.scale
 
 
-class DATETIME64:
-    """Class for holding `datetime64` value information in Firebolt DB."""
-
-    _prefix = "DateTime64("
-
-    def __init__(self, precision: int):
-        self.precision = precision
-
-    def __str__(self) -> str:
-        return f"DateTime64({self.precision})"
-
-    def __hash__(self) -> int:
-        return hash(str(self))
-
-    def __eq__(self, other: object) -> bool:
-        if not isinstance(other, DATETIME64):
-            return NotImplemented
-        return other.precision == self.precision
-
-
-NULLABLE_PREFIX = "Nullable("
+NULLABLE_SUFFIX = "null"
 
 
 class _InternalType(Enum):
     """Enum of all internal Firebolt types, except for `array`."""
 
-    # INT, INTEGER
-    Int8 = "Int8"
-    UInt8 = "UInt8"
-    Int16 = "Int16"
-    UInt16 = "UInt16"
-    Int32 = "Int32"
-    UInt32 = "UInt32"
-
-    # BIGINT, LONG
-    Int64 = "Int64"
-    UInt64 = "UInt64"
-
-    # FLOAT
-    Float32 = "Float32"
-
-    # DOUBLE, DOUBLE PRECISION
-    Float64 = "Float64"
+    Int = "int"
+    Long = "long"
+    Float = "float"
+    Double = "double"
 
-    # VARCHAR, TEXT, STRING
-    String = "String"
+    Text = "text"
 
-    # DATE
-    Date = "Date"
-    Date32 = "Date32"
-    PGDate = "PGDate"
+    Date = "date"
+    DateExt = "date_ext"
+    PGDate = "pgdate"
 
-    # DATETIME, TIMESTAMP
-    DateTime = "DateTime"
-    TimestampNtz = "TimestampNtz"
-    TimestampTz = "TimestampTz"
+    Timestamp = "timestamp"
+    TimestampExt = "timestamp_ext"
```

Comment:
I think Mosha said `timestamp_ext` is no longer supported. Might as well get rid of it.

Reply:
A query like `CAST('2019-07-31 01:01:01.1234' AS TIMESTAMP_EXT(4))` still return `timestamp_ext` for me, I guess we`ll need to keep it for now